common/web/types/src/package/kps-file.ts

```diff
@@ -41,6 +41,7 @@ export interface KpsFileOptions {
   followKeyboardVersion?: string;
   readMeFile?: string;
   graphicFile?: string;
+  licenseFile?: string;
```

Comment:
Does this get added to https://github.com/keymanapp/api.keyman.com/blob/466072dec5816fefb98ae2df06f0bb6957086eaa/schemas/kps/7.0/kps.xsd#L28-L39 or another schema?

Reply:
Yes, copying the schemas over to api.keyman.com will come in a future PR once they have all stabilised. The changes are current in this PR's kps.xsd.